packages/framework/src/schemas/steps/channels/in-app.schema.ts

```diff
@@ -54,8 +54,14 @@ const actionSchema = {
 const inAppOutputSchema = {
   type: 'object',
   properties: {
-    subject: { type: 'string' },
-    body: { type: 'string' },
+    subject: {
+      type: 'string',
+      minLength: 1,
+    },
+    body: {
+      type: 'string',
+      minLength: 1,
+    },
```

Comment:
same here why do we need min, if it is already required.
this schema is not aligned with the Novu Control Schema, are we sanitizing those values in dashboardSanitizeControlValues so we will have both, if we do not framework output validation will throw error.

Reply:
1. We need min to avoid allowing the empty string as a valid body or subject value. Without min what's happening is that subject and body return a validation error only the first time before being set as controls. If you type something, save and then remove it completely, it's being store as an empty string that passes the required validation.

2. Framework schema and API in-app control schema are aligned. See [here](https://github.com/novuhq/novu/pull/8093/files). The sanitization happens in parallel with all control inputs. Which is the case that will throw an error?

Reply:
sounds good to me! 👍🏻